src/hypothesis/internal/renaming.py

```diff
@@ -1,78 +0,0 @@
-# coding=utf-8
-#
-# This file is part of Hypothesis, which may be found at
-# https://github.com/HypothesisWorks/hypothesis-python
-#
-# Most of this work is copyright (C) 2013-2018 David R. MacIver
-# (david@drmaciver.com), but it contains contributions by others. See
-# CONTRIBUTING.rst for a full list of people who may hold copyright, and
-# consult the git log if you need to determine who owns an individual
-# contribution.
-#
-# This Source Code Form is subject to the terms of the Mozilla Public License,
-# v. 2.0. If a copy of the MPL was not distributed with this file, You can
-# obtain one at http://mozilla.org/MPL/2.0/.
-#
-# END HEADER
-
-from __future__ import division, print_function, absolute_import
-
-from inspect import cleandoc
-
-from hypothesis._settings import note_deprecation
-from hypothesis.internal.reflection import proxies
-
-
-def renamed_arguments(**rename_mapping):
```

Comment:
Do we actually want to get rid of this? We may want to rename arguments in future.

Reply:
My 2 cents: given the issues described in https://github.com/HypothesisWorks/hypothesis-python/issues/1111, it makes sense to get rid of it for now.

Reply:
IMO it's better to resurrect it from the git history if or when when need it - ideally our API style guide will mean we don't need to rename things again, and we'd have to keep the tests around too.